app/widgets/common/plugins/PluginActions.ts

```diff
@@ -1,17 +1,22 @@
 import type { PostPluginsUploadQueryParams } from 'backend/routes/Plugins.types';
 
+export type ResourceName = 'wagon' | 'yaml' | 'icon';
+export type Resources = Record<
+    ResourceName,
+    {
+        url: string;
+        file: unknown;
```

Comment:
Optional:

Probably that'll also work:
```suggestion
        file: File | null;
```

Reply:
Yeah, works as expected 💪 
I've also modified L21, as it was also containing this type declaration for the `file`

Reply:
Applied within 66ab8c82bd0b8996b3bba665f4fe5cc0f469878c